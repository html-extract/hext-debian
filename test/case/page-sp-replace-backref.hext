<h1 @text:replace(/(\d+): (.*)/, "$2 ($1)"):head />
