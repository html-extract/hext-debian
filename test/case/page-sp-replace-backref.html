<h1>1: First heading</h1>
<h1>2: Second heading</h1>
<h1>3: Third heading</h1>
<h1>4: Fourth heading</h1>
<h1>5: Fifth heading</h1>
