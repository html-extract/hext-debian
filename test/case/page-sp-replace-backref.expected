{"head":"First heading (1)"}
{"head":"Second heading (2)"}
{"head":"Third heading (3)"}
{"head":"Fourth heading (4)"}
{"head":"Fifth heading (5)"}
